fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/behaviour/interactive-parts.xml

```diff
@@ -265,8 +265,18 @@
                 <SWITCH_POSITION_TYPE>L</SWITCH_POSITION_TYPE>
                 <SWITCH_POSITION_VAR>XMLVAR_SEAT_FWDAFT</SWITCH_POSITION_VAR>
                 <MOMENTARY_REPEAT_FREQUENCY>800</MOMENTARY_REPEAT_FREQUENCY>
-                <CODE_POS_0>(L:SEAT_CPT_MOVE_FWD_AFT) 0.4 + 100 min (&gt;L:SEAT_CPT_MOVE_FWD_AFT)</CODE_POS_0>
-                <CODE_POS_2>(L:SEAT_CPT_MOVE_FWD_AFT) 0.4 - 0 max (&gt;L:SEAT_CPT_MOVE_FWD_AFT)</CODE_POS_2>
+                <CODE_POS_0>(L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) if{
+                    (L:SEAT_CPT_MOVE_FWD_AFT) 0.4 + 100 min (&gt;L:SEAT_CPT_MOVE_FWD_AFT)
+                } els{
+                    (L:SEAT_CPT_MOVE_FWD_AFT)
```

Comment:
Why are the else statements needed? They are just reading an lvar onto the stack and then doing nothing with it.